kujira/store/events_queue.py

```diff
@@ -0,0 +1,42 @@
+'''events queue module'''
+from . import redis_db
+
+
+class RedisQueue(redis_db. RedisConnection):
+    '''management of tasks in Redis database '''
+    def __init__(self):
+        '''init'''
+        super(RedisQueue, self).__init__()
+        self.connection_adress = 'localhost'
```

Comment:
Cosntructor of base class should set attributes: connection_address and connection_port